README.md

```diff
@@ -63,6 +64,12 @@ perform pool operations e.g pool provisioning.
 **Note:** It is not recommended to modify the CSPI CR and pool-manager in the running cluster unless you know what you are 
 trying to do. CSPC should be the only point of interaction.
 
+Once the CStor pool get provisioned successfully, admin/user can create PVC to provision csi CStor volumes, when a user
+creates PVC, CStor CSI driver creates CStorVolumeConfig(CVC) resource, managed and reconcile by the **cvc-controller** which creates
+different volume specific resources for each persistent volume, later managed by their respective controllers, more info
```

Comment:
There is only one persistent volume for a PVC. So I think `each` is not required.

Reply:
We can delete this part `for each persistent volume`

Reply:
Here we are talking in reference to cvc-operator, which looks ok to me